syncd/FlexCounter.cpp

```diff
@@ -511,8 +519,72 @@ class CounterContext : public BaseCounterContext
         }
     }
 
+    void addVoqEntryToChassisAppDb(_In_ const sai_object_id_t rid,
+                                   _In_ const sai_object_id_t vid,
+                                   _In_ std::vector<swss::FieldValueTuple> &counterValues,
+                                   _Out_ swss::Table *appDbCountersTable ) {
```

Comment:
{ to new line, plese forllow code quality

Reply:
Done.